lib/heuristics/scheduling_heuristic.rb

```diff
@@ -63,6 +63,7 @@ def initialize(vrp, job = nil)
       @uninserted = {}
       @missing_visits = {}
 
+      @wrapper_tool = Wrappers::Wrapper.new
```

Comment:
I am not sure if we need to generate a new Wrapper object in fact any  `OptimizerWrapper.config[:services][service]` object would do.

We can name the variable simply `@wrapper`, with `tool` it sounds as if it is something different.